scripts/test-truncating-of-client-hello.py

```diff
@@ -20,10 +19,10 @@
 from tlslite.constants import CipherSuite, AlertLevel, AlertDescription, \
         ExtensionType
 from tlslite.extensions import TLSExtension
```

Comment:
`TLSExtension` looks unused.

Reply:
(when we are in the modernizing and cleaning up)

Reply:
removed